conf/services/direct-debit.cy.yml

```diff
@@ -22,52 +22,52 @@ milestones:
       yn bodloni maen prawf llwyddiant 1.3.5 (Identify Input Purpose) Canllawiau
       Hygyrchedd Cynnwys y We.
-    date: 2020-07-31
+    date: 2021-12-15
   - description: |
       Mae gan y gwasanaeth wallau yn nhrefn yr elfennau, y cystrawen priodoli, ac mae’r
       tagiau dechrau a diwedd wedi’u trefnu’n anghywir sy’n golygu nad yw’r
       gwasanaeth yn ddibynadwy i dechnoleg gynorthwyol. Nid yw hyn yn bodloni maen
       prawf llwyddiant 4.1.1 (Parsing) Canllawiau Hygyrchedd Cynnwys y We.
-    date: 2020-09-30
+    date: 2021-12-15
   - description: |
       Nid yw’r cysylltiadau ‘newid’ ar y dudalen ‘Gwirio bod eich manylion yn gywir’
       yn ddigon disgrifiadol, ac mae ganddynt yr un enw. Nid yw hyn yn bodloni maen
       prawf llwyddiant 2.4.4 (Link Purpose (Link only)) Canllawiau Hygyrchedd
       Cynnwys y We, fersiwn 2.1
-    date: 2020-09-30
+    date: 2021-12-15
   - description: |
       Ar y dudalen ‘Dilysu’ch cyfeiriad e-bost’, mae cynnwys cudd a allai achosi
       problemau i’r rhai sy’n defnyddio darllenydd sgrin. Nid yw hyn yn bodloni
       maen prawf llwyddiant 4.1.2 (Name, Role, Value) Canllawiau Hygyrchedd
       Cynnwys y We.
-    date: 2020-09-30
+    date: 2021-12-15
   - description: |
       Ar y dudalen ‘Pa e-bost ydych chi am ei ddefnyddio?’, mae allwedd ar goll,
       felly gall pwrpas y botymau radio fod yn aneglur i rai defnyddwyr.
       Nid yw hyn yn bodloni maen prawf llwyddiant 1.3.1 (Info and Relationships)
       Canllawiau Hygyrchedd Cynnwys y We.
-    date: 2020-09-30
+    date: 2021-12-15
   - description: |
       Mae’r gwasanaeth yn dod â sesiwn y defnyddiwr i ben os ydyw wedi bod yn
       segur am 15 munud, ond nid yw’n rhoi cyfle i’r defnyddiwr ymestyn ei sesiwn.
       Nid yw hyn yn bodloni maen prawf llwyddiant 2.2.1 (Timing Adjustable)
       Canllawiau Hygyrchedd Cynnwys y We.
-    date: 2020-09-30
+    date: 2021-12-15
   - description: |
       Nid yw’r testun awgrym ar y maes ‘Nodwch y cyfeiriad e-bost newydd’ yn
       gysylltiedig â’r maes dan sylw. Mae hyn yn golygu na fydd defnyddiwr
       darllenydd sgrin yn clywed y testun awgrym wrth symud yn uniongyrchol
       i’r mewnbynnau radio. Nid yw hyn yn bodloni maen prawf llwyddiant 2.4.6
       (Headings and Labels) Canllawiau Hygyrchedd Cynnwys y We.
-    date: 2020-09-30
+    date: 2021-12-15
   - description: |
       Ar y dudalen ‘Dilysu’ch cyfeiriad e-bost’, defnyddir botwm wrth ddangos
       y cysylltiad ‘Anfonwch eto’. Mae hyn yn golygu bod y rhyngweithio’n
       wahanol i’r hyn a ddywedir wrth y defnyddiwr, naill ai’n weledol neu
       drwy ddarllenydd sgrin. Nid oes gan y botwm hwn nodwedd ffocysu sy’n
       golygu nad yw’n gydnaws â llywio drwy fysellfwrdd. Nid yw hyn yn bodloni
       maen prawf llwyddiant 2.4.7 (Focus Visible) Canllawiau Hygyrchedd Cynnwys y We.
-    date: 2020-09-30
+    date: 2021-12-15
 serviceLastTestedDate: 2020-07-06
 statementVisibility: public
 statementCreatedDate: 2020-06-30
```

Comment:
Does the `statementLastUpdatedDate` need to be updated here?

Reply:
Yes